lab-justin/route/task-router.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const Router = require('express').Router;
+const debug = require('debug')('justin:task-router');
+const jsonParser = require('body-parser').json();
+const taskCrud = require('../lib/task-crud');
+const taskRouter = module.exports = new Router();
+
+debug('justin:task-router');
+
+taskRouter.post('/note/:id/task', jsonParser, function(req, res){
+  taskCrud.createTask(req.body)
+  .then( task => res.send(task))
+  .catch( err => res.sendError(err));
+});
+
+taskRouter.get('/note/:id/task', function(req, res){
```

Comment:
This is sort of weird, because you're using the id from the url, but the url seems like it should be a note's id, not a tasks. I would set up my url like '/note/:noteId/task/:taskId'